wellcomeml/ml/spacy_classifier.py

```diff
@@ -112,6 +114,11 @@ def yield_train_data(X_train, Y_train):
             optimizer = self.nlp.begin_training()
             optimizer.alpha = self.learning_rate
             #optimizer.L2 = 1e-4
+
+            if self.pre_trained_vectors:
```

Comment:
might be good to name `pre_trained_vectors` so it's clear it is a path type?

Reply:
what is the name suggestion? `pre_trained_vectors_path`?

Reply:
sure!